client/src/pages/GameThread.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import Header from "../components/Header";
+import "../styles/GameThread.css";
+
+class GameThread extends React.Component {
+  render() {
+    return (
+      <div className="weirdContainer">
+        {/* <Header /> */}
+        <div className="Title">PUT GAMETHREAD HERE</div>
```

Comment:
This should be a lowercase "title" className to be consistent

Reply:
thanks!